src/jit/gentree.cpp

```diff
@@ -764,6 +764,59 @@ bool GenTree::gtHasReg() const
     return hasReg;
 }
 
+//-----------------------------------------------------------------------------
+// GetRegisterDstCount: Get the number of registers defined by the node.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    Reg Mask of GenTree node.
+//
+// Notes:
+//    This should not be called on a contained node.
+//
+int GenTree::GetRegisterDstCount() const
+{
+    assert(!isContained());
+    if (!IsMultiRegNode())
+    {
+        return (IsValue()) ? 1 : 0;
+    }
+    if (IsMultiRegCall())
```

Comment:
Might be nice to make it `else if` to be consistent with the rest of the code. Or turn all the following `else if` into `if`.